src/customize-pop/gnome-tweaks-extensions/user-themes.md

```diff
@@ -0,0 +1,94 @@
+# Managing User Themes
+
+User themes include changes to window animations, desktop color schemes, window designs, icons, and workspaces. You can install custom GNOME Themes from opendesktop.org. The Pop!\_OS default themes can be re-applied using GNOME Tweaks or the by using Terminal commands.
```

Comment:
> You can install custom GNOME Themes from opendesktop.org.

First, it's probably good to include a hyperlink if you're going to mention a URL like that.

OpenDesktop.org has a number of services (including what looks like a Nextcloud instance) in addition to just themes. Themes are located specifically on [Pling](https://www.pling.com/), or [Gnome-look.org](https://www.gnome-look.org) for only GNOME themes. I would recommend mentioning/linking to one of those instead.

Also, this is just one third-party platform for sharing themes. I would add "or other websites" or "or other sources" or similar verbiage after the link.